addons/point_of_sale/static/src/app/pos_app.scss

```diff
@@ -10,6 +10,7 @@
        -moz-user-select: none;
             user-select: none;
     text-shadow: none;
+    touch-action: manipulation;
```

Comment:
wouldn't it be better to disable zooming completely ? Even pinch to zoom

Reply:
> wouldn't it be better to disable zooming completely ? Even pinch to zoom

I think that's a bad UX. Some people (including me) tend to pinch zoom.

Reply:
@vlst-odoo @caburj Do not restrict the pinch zoom